gibberish_server/controllers/room_controller.js

```diff
@@ -4,14 +4,13 @@ const roomModel = require("../models/Room")
 
 class RoomController {
     static getRoomData(req, res) {
-        let r = roomModel.getRoomInfoAsObject(req.params["roomId"]);
-        r.nextState();
```

Comment:
you should not delete this

Reply:
yup, i have added it back in the latest commits. thanks for letting me know what nextState() does